packages/appstore/src/components/cfds-listing/index.tsx

```diff
@@ -14,6 +13,7 @@ import { Actions } from 'Components/containers/trading-app-card-actions';
 import { getHasDivider } from 'Constants/utils';
 import { AvailableAccount, TDetailsOfEachMT5Loginid } from 'Types';
 import './cfds-listing.scss';
+import { useCFDCanGetMoreMT5Accounts } from '@deriv/hooks';
 
 type TDetailedExistingAccount = AvailableAccount &
     TDetailsOfEachMT5Loginid &
```

Comment:
@shontzu-deriv check if this type definition is being used in the Webtrader_url card!
<img width="1347" alt="Screenshot 2023-10-10 at 2 23 36 PM" src="https://github.com/binary-com/deriv-app/assets/120543468/7fd81cf2-107f-4678-b00e-f84e2ffc416f">

Reply:
I'll check it with @shontzu-deriv.

Reply:
interesting, as the webtrader_url card is not yet merged, I think it is safe to remove it since it will be added back with the release of the mt5 webtrader_url card @hamza-deriv ?

Reply:
Sure thanks for checking